lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+class Menu
+  attr_reader :dishes
+
+  DISHES = { 'hawaian pizza' => 6,
+             'beef noodles' => 5,
+             'chicken tikka massala' => 6,
+             'pepperoni pizza' => 5 }
+
+  def initialize(dishes = DISHES)
+    @dishes = dishes
+  end
+
+  def display
+    @dishes.map { |k, v| "#{k}: £#{v}" }
```

Comment:
Would it work to use different names for the key and value here. Just to make it clear what they refer to. Alternatively, you could add what they refer to in a comment above.